script.js

```diff
@@ -0,0 +1,94 @@
+
+var disp = $("#display")[0]; // Needed to return native DOM object.
+var d = disp.getContext("2d");
+
+var TILE_SIZE = 200;
+
+var curPlayer = "X";
+var curColor = "rgba(255,0,0,0.25)";
```

Comment:
would look into SASS, or an css precompiler, that helps with dynamic values. Allows you to create variables, write methods, etc...